README.md

```diff
@@ -1,7 +1,55 @@
-# Gruvbox Theme
+
+<h1 align="center">
+  <br>
+  <a href="https://marketplace.visualstudio.com/items?itemName=jdinhlife.gruvbox">
+  <img src="images/icon.png">
+  </a>
+  <br>
+  Gruvbox
+  <br>
+</h1>
+
+<h4 align="center">A port of Gruvbox theme to VS Code editor
+</h4>
+
+<p align="center">
+  <a href="https://marketplace.visualstudio.com/items?itemName=jdinhlife.gruvbox">
+    <img src="https://vsmarketplacebadge.apphb.com/version-short/jdinhlife.gruvbox.svg?style=flat-square" alt="Version">
+  </a>
+  <a href="https://marketplace.visualstudio.com/items?itemName=jdinhlife.gruvbox">
+    <img src="https://vsmarketplacebadge.apphb.com/installs/jdinhlife.gruvbox.svg?style=flat-square" alt="Installs">
+  </a>
+  <a href="https://marketplace.visualstudio.com/items?itemName=jdinhlife.gruvbox">
+      <img src="https://vsmarketplacebadge.apphb.com/rating/jdinhlife.gruvbox.svg?style=flat-square" alt="Ratings">
+  </a>
+</p>
+
+<p align="center">
+  <a href="#about">About</a> |
+  <a href="#installation">Installation</a> |
+  <a href="#variants">Variants</a> |
+  <a href="#contributing">Contributing</a> |
+  <a href="#release-notes">Release Notes</a> | 
+  <a href="#credits">Credits</a>
+</p>
```

Comment:
Is it possible to make these all markdown? Would make things a bit easier to read.

I'm alright with not having those center-aligned stuff as long as you're ok with it.
If it's not possible, it's alright though

Reply:
@jdinhify hi! I use this template in my projects, but I can change it. 

I could not convert the icon in markdown (change position made the REAME ugly), but I convert menu in markdown. What do you think? See here: ttps://github.com/mstuttgart/vscode-theme-gruvbox/tree/add-icons

Reply:
Thanks, let's use this. Not having menu is ok since the README is small anyway. I'll merge this PR and deploy soon when I have time.